whereby/sdk
# React native webview example app

### Description

Adds a react native example app, that uses a webview to render an embedded whereby room.

**Summary:**



**Related Issue:**



### Testing

1. Test with PWA PR https://github.com/whereby/pwa/pull/4185
2. Replace `apps/react-native-webview-example/app/room/index.tsx` line 5 with your local stack room.
3. Start the react native app with either `yarn dev:react-native-webview-example` in root folder, or cd into `apps/react-native-webview-example` and run `yarn dev`
4. Start a simulator, join the room and verify that you see the event log in the top (should list all incoming events)

### Screenshots/GIFs (if applicable)
![image](https://github.com/user-attachments/assets/065dc3c8-538f-4f71-8d04-12661b34ad85)



### Checklist

        under the terms of the MIT license.

### Additional Information
